test/mocha.opts

```diff
@@ -1 +0,0 @@
---timeout 60000
```

Comment:
My understanding of mocha is that it looks for this file in the root of `/test` so it seemed to not have any effect after I moved the test files to a new directory.

I added the [config option](https://github.com/18F/federalist/blob/pull/new-post/package.json#L73) to the `package.json` so that the test doesn't timeout on Travis. Would it be better to keep this file and pass in the path to `mocha`?